imap_processing/idex/idex_l1a.py

```diff
@@ -108,7 +108,7 @@ class PacketParser:
     .. code-block:: python
 
         from imap_processing.idex.idex_l1a import PacketParser
-        l0_file = "imap_processing/tests/idex/imap_idex_l0_sci_20230725_v001.pkts"
+        l0_file = "imap_processing/tests/idex/imap_idex_l0_sci_12142023_v001.pkts"
```

Comment:
can you change date format of the file to match `yyyymmdd`?

Reply:
Yes - good catch!